### ISSUE TITLE
The plugin throws an exception after update on Xcode10

### ISSUE THREAD

**Issue by mykola-mokhnach** [lyndsey-ferguson/fastlane-plugin-test_center#174]
### New Issue Checklist

- [x] Updated `fastlane-plugin-test_center` to the latest version
- [x] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [x] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [x] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

### Issue Description
After updating fastlane and test center plugin to the most recent version our testing script on Travis started failing with the following stack trace:

```
Found no similar issues. To create a new issue, please visit:
2001https://github.com/fastlane/fastlane/issues/new
2002Run `fastlane env` to append the fastlane environment to your issue
2003bundler: failed to load command: fastlane (/Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/bin/fastlane)
2004NameError: [!] uninitialized constant Fastlane::Actions::MultiScanAction::ScanHelper
2005Did you mean?  Fastlane::Helper
2006  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-plugin-test_center-3.8.13/lib/fastlane/plugin/test_center/actions/multi_scan.rb:140:in `use_scanfile_to_override_settings'
2007  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-plugin-test_center-3.8.13/lib/fastlane/plugin/test_center/actions/multi_scan.rb:116:in `prepare_for_testing'
2008  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-plugin-test_center-3.8.13/lib/fastlane/plugin/test_center/actions/multi_scan.rb:16:in `run'
2009  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/runner.rb:256:in `block (2 levels) in execute_action'
2010  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
2011  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/runner.rb:235:in `block in execute_action'
2012  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/runner.rb:227:in `chdir'
2013  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/runner.rb:227:in `execute_action'
2014  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/runner.rb:157:in `trigger_action_by_name'
2015  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/fast_file.rb:159:in `method_missing'
2016  ../fastlane/Fastfile:20:in `block in parsing_binding'
2017  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/lane.rb:33:in `call'
2018  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
2019  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'
2020  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/runner.rb:45:in `execute'
2021  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/lane_manager.rb:56:in `cruise_lane'
2022  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle'
2023  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/commands_generator.rb:108:in `block (2 levels) in run'
2024  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
2025  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
2026  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
2027  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
2028  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
2029  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/commands_generator.rb:349:in `run'
2030  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/commands_generator.rb:41:in `start'
2031  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/fastlane/lib/fastlane/cli_tools_distributor.rb:119:in `take_off'
2032  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/gems/fastlane-2.136.0/bin/fastlane:23:in `<top (required)>'
2033  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/bin/fastlane:23:in `load'
2034  /Users/travis/build/appium/WebDriverAgent/vendor/bundle/ruby/2.6.0/bin/fastlane:23:in `<top (required)>'
```

Please find the full log at https://travis-ci.org/appium/WebDriverAgent/jobs/614495760

The problem is only reproducible with Xcode10 SDK. See https://github.com/appium/WebDriverAgent/pull/252 for more details

**Comment by lyndsey-ferguson** [comment]
This does not fail with Xcode 11?

**Comment by mykola-mokhnach** [comment]
Yes, Xcode11 is totally fine as you can see in https://travis-ci.org/appium/WebDriverAgent/builds/614495733

Also, it worked properly with both Xcode version before I updated the plugin and Fastlane

**Comment by lyndsey-ferguson** [comment]
Ug, I don't know what I'm looking at for the multi_scan logs.

**Comment by mykola-mokhnach** [comment]
Do you mean the [log](https://travis-ci.org/appium/WebDriverAgent/jobs/614495760) link above or which link would you like to have?

**Comment by mykola-mokhnach** [comment]
The fast lane script is located at https://github.com/appium/WebDriverAgent/blob/master/Fastlane/Fastfile

**Comment by lyndsey-ferguson** [comment]
Can you link to the log where it works fine with Xcode 11?

What version of the test_center plugin were you using before?

**Comment by mykola-mokhnach** [comment]
Sure, here is the log where the previous version properly works with Xcode 10: https://travis-ci.org/appium/WebDriverAgent/jobs/612418172

The version change is visible here: 
https://github.com/appium/WebDriverAgent/pull/252/files#diff-e79a60dc6b85309ae70a6ea8261eaf95R68 (line 68) -> fastlane-plugin-test_center (3.6.3) to fastlane-plugin-test_center (3.8.13)

**Comment by lyndsey-ferguson** [comment]
@mykola-mokhnach can you modify your `Pluginfile` per my instructions below, run `bundle install`, and then run your fastlane again (with the `--verbose` flag)?

Pluginfile:
```
gem 'fastlane-plugin-test_center', :git => "https://github.com/lyndsey-ferguson/fastlane-plugin-test_center.git", :branch => "issue-174-uninitialized-constant"
```

If there are still problems, please let me know and attach the console output as a text file to this issue (makes it easier for me to review).  If it works, please let me know.

**Comment by mykola-mokhnach** [comment]
Unfortunately the job still fails. Please find the verbose log at https://travis-ci.org/appium/WebDriverAgent/jobs/615413571

Xcode11 run is still fine though: https://travis-ci.org/appium/WebDriverAgent/jobs/615413563

**Comment by mykola-mokhnach** [comment]
Attaching the raw text log as requested: 
[log.txt](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/files/3878795/log.txt)

**Comment by lyndsey-ferguson** [comment]
I've made some changes. Can you try updating your `Gemfile.lock` to use this git SHA: `c0ba3dbaa19e234f3f59f3ea744a815acea683f1`, bundle install, and re-run your lane?

**Comment by lyndsey-ferguson** [comment]
@mykola-mokhnach have you been able to test this yet? Someone else experiencing something similar has confirmed that my fix above solves their issue.

**Comment by mykola-mokhnach** [comment]
sorry, Travis CI is not the fastest one :)

Yes, with the recent commit I can confirm the bug is fixed and the plugin works for both Xcode11 and Xcode10:
- 11: https://travis-ci.org/appium/WebDriverAgent/jobs/615647128
- 10: https://travis-ci.org/appium/WebDriverAgent/jobs/615647136

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.8.15](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.8.15)
